workflows/contrib/start_service.py

```diff
@@ -1,4 +1,4 @@
-from optparse import SUPPRESS_HELP, OptionParser
+import argparse
```

Comment:
That breaks the existing API, so will require associated changes in dlstbx (https://github.com/DiamondLightSource/python-dlstbx/blob/161148de9043c3ca51bd28556075014711707f71/src/dlstbx/cli/service.py#L79-L80)
and zocalo
(https://github.com/DiamondLightSource/python-zocalo/blob/79d13c36b71cfec7bd6f9619bb0d92c66f5ea62e/zocalo/service/__init__.py#L71-L72)